pkg/storage/objects/store.go

```diff
@@ -136,16 +136,16 @@ func (o *objStore) List(ctx context.Context, prefix, startAfter, endBefore storj
 	for i, itm := range strItems {
 		items[i] = ListItem{
 			Path:     itm.Path,
-			Meta:     convertMeta(itm.Meta),
+			Meta:     ConvertMeta(itm.Meta),
 			IsPrefix: itm.IsPrefix,
 		}
 	}
 
 	return items, more, nil
 }
 
-// convertMeta converts stream metadata to object metadata
-func convertMeta(m streams.Meta) Meta {
+// ConvertMeta converts stream metadata to object metadata
+func ConvertMeta(m streams.Meta) Meta {
```

Comment:
Would be nice if we could take this opportunity to rename this function to something like `StreamMetaToObjectMeta()`. Unless you think there's already enough going on and we should strictly open new PRs for that sort of thing.